ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/CCodeGenerator_Test.java

```diff
@@ -683,6 +683,16 @@ public void testEmbeddedTypesSet() {
         assertEquals(embeddedTypes.size(), 3);
         assertTrue(embeddedTypes.contains("ds3_string_response"));
     }
+
+    @Test
+    public void testPaginatedTypesSet() {
+        final Ds3ApiSpec getBucketsSpec = new Ds3ApiSpec(
+                ImmutableList.of(Ds3ModelFixtures.getBucketsSpectraS3Request()),
```

Comment:
This test would be more complete if you also had a non-paginated request to verify that the filter is working properly